src/Kernel32.Tests.Shared/NTStatusExceptionFacts.cs

```diff
@@ -54,10 +58,11 @@ public void NTStatusException_Informational_Message()
     }
 
     [Fact]
+    [UseCulture("en-US")]
```

Comment:
The way it is now, the thread `CurrentCulture` controls how `FormatMessge` localizes the message. It makes more sense to have it matching the current thread culture instead of being a hard-coded value or the default system language.